HomeTask3/HomeTask3/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace HomeTask3
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите значения аргумента для функции f(x)");
+            var argument = double.Parse(Console.ReadLine());
+            if ((argument == 0) || (argument < 1))
```

Comment:
Условие `argument == 0` лишнее, так как если `argument` равен 0, то условие `argument < 1` будет иметь значение `true`
```suggestion
            if (argument < 1)
```